source/cloud-native-platform/guides/Migrate-Azure-Front-Door-to-standard-or-Premium-tier.html.md.erb

```diff
@@ -0,0 +1,22 @@
+---
+title: Migrate Azure Front Door (classic) to Standard/Premium tier
+last_reviewed_on: 2023-06-30
+review_in: 6 months
+weight: 10
+hide_in_navigation: true
+---
+
+# <%= current_page.data.title %>
+
+The Azure Portal can be used to perform an migration of a Azure Front door (Classic) to Standard/Premium tier
+
+### Prerequisite 
+
+- Review the [About Front Door tier migration](https://learn.microsoft.com/en-us/azure/frontdoor/tier-migration) article.
```

Comment:
Are we getting teams to migrate Frontdoor & not PlatOps?

Could we include an example migration / references to PRs links etc.

Reply:
sorry yes it will be PlatOps doing the migration,  am I better putting this guide in another repo?